wire-ios-transport/Source/Authentication/ZMAccessTokenHandler.m

```diff
@@ -304,7 +300,6 @@ - (BOOL)processAccessTokenResponse:(ZMTransportResponse *)response
         
         if ( ! didFail) {
             self.accessToken = newToken;
-            ZMLogInfo(@"New access token <%@: %p>", self.accessToken.class, self.accessToken);
```

Comment:
should we keep this info for debug build (around #if DEBUG macro with WireLogger) ?